pages/management/applications.md

```diff
@@ -4,76 +4,121 @@ title: Applications
+
+Once you push the code to your resin.io application, a **Docker** container will be built on our servers and downloaded to any online devices.
 
 For more details on deployment, check out our [deployment guide](/deployment/deployment).
 
-## Application Actions
+## Device filters
+
+As the number of devices in your application grows, the device list will become increasingly busy. Filters provide a convenient way to quickly find specific devices based on shared characteristics. 
+
+### Add and update filters
+
+To add a filter, click the *Add filter* button near the top-left corner of your application page. You'll be presented with a window to configure your filter:
+
+<img src="/img/common/app/add-filter.png" width="80%">
+
+The first dropdown lists the device characteristics you can filter on. These characteristics correspond to the columns in your device list. The second dropdown contains comparison operators appropriate to the device characteristic, such as `is`, `is before`, `contains`, and `matches RegEx`. The third field is where you specify a value to filter on. Click *Add filter* to apply your configuration.
+
+When your filter has been added, you will see it in a box above the device list. The device list will now only show devices that match your filter:
+
+<img src="/img/common/app/filter-applied.png" width="80%">
+
+If you need to update your filter, simply click it and a new configuration window will appear. To remove, click the `x` to the filter's right. You can add any number of additional filters with the *Add filter* button.
+
+### Create a view
 
-### Download Image
+When you create a view, you are saving a specific set of filters that you may want to use again. To do this, click *Save view* on the right side of the filter box:
 
-This action will allow you to download a new device OS image so you can provision new devices into your fleet. It will also ask you to select and configure you network preferences before you download.
+<img src="/img/common/app/save-view.png" width="80%">
 
-### Restart Application (on all devices)
+You will be asked to choose a name for the saved view. This view can then be selected at any time by clicking the *Views* button above the filter box.
 
-The `Restart Application` action is a fleet wide action that will restart the application container on all the devices that are currently online. It should be noted that currently these action notifications are not queued up, so if a device is offline when the action is triggered, it will never be notified of it.
+## Application actions
 
-## Dangerous Application Actions
+Actions let you change the state of some or all of the devices in your application. They can be applied in two ways:
 
-### Purge Data (on all devices)
+1. The *Actions* menu, located on the left side of the application page, allows you to apply state changes to all devices in the application.
```

Comment:
A product question for @shaunmulligan - is it superfluous (and maybe even confusing) to have this on the application page, when we also have Group Actions to the top right of the device list, which could easily be applied to your entire fleet? Should we combine the two somehow? Or remove any confusing overlap? Right now it's not clear where you should go first to apply an action to your entire fleet. 

The actions that appear in both places are: restart app, reboot, purge, shutdown. 
The actions that are unique to the actions page are: public url, support access, remove collaborators